Gemfile

```diff
@@ -34,6 +34,7 @@ group :production do
   gem 'rails_12factor'
 end
 
+gem 'json'
```

Comment:
I believe rails itself supports JSON, but should be okay to add.

Reply:
Oh travis was complaining when I didn't have it, but was fine after I added it haha